lms/static/js/verify_student/models/reverification_model.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Model for a reverification attempt.
+ *
+ * The re-verification model is responsible for
+ * storing face photo image data and submitting
+ * it back to the server.
+ */
+var edx = edx || {};
+
+(function( $, _, Backbone ) {
+    'use strict';
+
+    edx.verify_student = edx.verify_student || {};
+
+    edx.verify_student.ReverificationModel = Backbone.Model.extend({
+
+        defaults: {
+            courseKey: '',
+            checkpointName: '',
+            faceImage: '',
+        },
+
+        sync: function( method, model ) {
```

Comment:
We shouldn't have to pass the model into the sync method. We can just use this to access the model (e.g. this.get( 'faceImage' )